examples/server/utils.hpp

```diff
@@ -650,11 +650,15 @@ static json format_final_response_oaicompat(const json & request, const json & r
         res["completion_probabilities"] = json_value(result, "completion_probabilities", json::array());
     }
 
+    if (timings) {
```

Comment:
```suggestion
    if (result.contains("timings")) {
```

Reply:
Do you mean `request.contains("timing_per_token")` ?

Reply:
no, `result.contains("timings")`

if you print out the `result` object, you will understand

Reply:
If I understand correctly, `result` is assigned to `res.data` in `server.cpp`. In `final_response`, `result.contains("timings")` is always true. 
Do you mean that in `final_response`, the `timing_per_token` parameter is ignored and `timings` are always returned, while this parameter is only effective in `partial_response`? Or is it determined based on whether the `timing_per_token` parameter is set to true to decide whether to include `timings` in `res.data`?

Reply:
`send_partial_response` must only set `timings` object when `slot.timing_per_token` is set

Otherwise, what you're doing make no sense because `timings` is always calculated even when user does not set `timing_per_token`